tests/vmi_multus_test.go

```diff
@@ -153,22 +154,23 @@ var _ = Describe("[Serial]Multus", func() {
 		result = virtClient.RestClient().
 			Post().
 			RequestURI(fmt.Sprintf(postUrl, tests.NamespaceTestDefault, "ptp-conf-1")).
-			Body([]byte(fmt.Sprintf(ptpConfCRD, "ptp-conf-1", tests.NamespaceTestDefault))).
+			Body([]byte(fmt.Sprintf(ptpConfCRD, "ptp-conf-1", tests.NamespaceTestDefault, ptpSubnet))).
 			Do()
 		Expect(result.Error()).NotTo(HaveOccurred())
 
 		result = virtClient.RestClient().
 			Post().
 			RequestURI(fmt.Sprintf(postUrl, tests.NamespaceTestAlternative, "ptp-conf-2")).
-			Body([]byte(fmt.Sprintf(ptpConfCRD, "ptp-conf-2", tests.NamespaceTestAlternative))).
+			Body([]byte(fmt.Sprintf(ptpConfCRD, "ptp-conf-2", tests.NamespaceTestAlternative, ptpSubnet))).
 			Do()
 		Expect(result.Error()).NotTo(HaveOccurred())
 	})
 
 	Describe("[rfe_id:694][crit:medium][vendor:cnv-qe@redhat.com][level:component]VirtualMachineInstance using different types of interfaces.", func() {
+		ptpGateway := libnet.NextIPAsString(cidrToIP(ptpSubnet))
```

Comment:
I do not understand the need for this. The test is specific, we are suppose to know what are the IP addresses we assign and what are the defaults of the IPAM used.

Reply:
> Extracting the subnet used in the tests to a const is nice, but incrementing an IP based on it is an overkill and usually not needed for tests (where we know what to expect).

So, do you suggest to use two different consts, one for the subnet, and one for the gateway?


> Taking that and the fact that we need to import a bunch of dependencies, is a sign we better avoid it.

Originally I just copy-pasted the relevant methods from the ptp repo, and used them, so no dependencies were added. 
After @RamLavi  question [here](https://github.com/kubevirt/kubevirt/pull/4387#issuecomment-736620915), I looked into it and saw [this stackoverflow](https://stackoverflow.com/questions/43148238/golang-import-only-one-file-from-package/43148312#43148312), and this [official doc](https://blog.golang.org/go1.7-binary-size), saying that when compiling it will import only the methods that are being used, and thought that the cost is not that high.

Reply:
> So, do you suggest to use two different consts, one for the subnet, and one for the gateway?

Yes

> Originally I just copy-pasted the relevant methods from the ptp repo, and used them, so no dependencies were added.
> After @RamLavi question [here](https://github.com/kubevirt/kubevirt/pull/4387#issuecomment-736620915), I looked into it and saw [this stackoverflow](https://stackoverflow.com/questions/43148238/golang-import-only-one-file-from-package/43148312#43148312), and this [official doc](https://blog.golang.org/go1.7-binary-size), saying that when compiling it will import only the methods that are being used, and thought that the cost is not that high.

The import has implications beyond the size. I see no justification here to calculate the next IP address.

Reply:
OK. Done.